### ISSUE TITLE
static build folder admin not relative 

### ISSUE THREAD

**Issue by theluk** [webiny/webiny-js#568]
### Question

hey there, 

I am doing some good progress, but currently I am hanging there, not sure what the solution might be. 

so I am running the admin and site via nginx proxy pass over one single domain, lets say `http://local.my.app`. so as soon as I use `http://local.my.app/admin` he correctly proxies the request to the container where admin site is running. Now the bad part is, that even though the admin root is "admin", the static scripts are tried to by fetched from the root of the directory. So the admin app tries to fetch `http://local.my.app/static/build....js`. There is of course no access to it and also it may conflict with my own static builds and also with the static builds of the `site` package.

I hope/am sure you have already a solution for that, maybe some simple config in rescript or somthing

Cheers and thanks a lot

**Comment by Pavel910** [comment]
@theluk this is a simple one :) All you have to do is set the environment variable to `PUBLIC_URL=/admin` in the admin app. It is part of the `create-react-app` configuration, actually has nothing to do with Webiny. That will add the `/admin` prefix to all assets generated by webpack. Let me know how it goes :)

**Comment by theluk** [comment]
ha, thanks a lot. Seems not to be working though - PUBLIC_URL seems to be ignored...

**Comment by Pavel910** [comment]
Yes, it is not meant to be used in development, only when building for production (I thought that's what's bothering you, sorry).

We ourselves work with local ports, I guess that's how CRA is meant to be used. But I can see that PR is almost live, so hang on just a little longer :)
